### PR TITLE
Updated dependencies and fixed an issue with Capped Collections

### PR DESCRIPTION
**Pull request by GeneralXception**

### CODE REVIEW

**Comment by nblumhardt** [comment]
Thanks for the PR!

The build is failing because the CSPROJ file's reference path hasn't been updated to match the version of the new Serilog package you've upgraded to.

Otherwise LGTM, thanks!

File: src/Serilog.Sinks.MongoDB/Sinks/MongoDB/MongoDBSink.cs [hunk 1]
```diff
@@ -82,6 +82,16 @@ public MongoDBSink(IMongoDatabase database, int batchPostingLimit, TimeSpan peri
             _collectionName = collectionName;
             _collectionCreationOptions = collectionCreationOptions;
             _formatProvider = formatProvider;
+
+            //  capped collections have to be created because GetCollection doesn't offer the option to create one implicitly
+            //  only create one if it doesn't already exist
+            if (_collectionCreationOptions.Capped.GetValueOrDefault(false)) {
```
Comment (nblumhardt) [inline src/Serilog.Sinks.MongoDB/Sinks/MongoDB/MongoDBSink.cs]: Would be nice to put `{`s on new lines to match the coding style in the rest of this file.

**Comment by nblumhardt** [comment]
@GeneralXception any update? Just received: https://github.com/serilog/serilog/issues/665

**Comment by GeneralXception** [comment]
Hi @nblumhardt , I've made the changes to the csproj to the correct version and the code style changes but the build is still broken. Any ideas?

**Comment by nblumhardt** [comment]
Pefect, thanks!

**Pull request merged**